api/forum/models.py

```diff
@@ -1,3 +1,40 @@
 from django.db import models
+from accounts.models import Profile
 
-# Create your models here.
+
+class Post(models.Model):
```

Comment:
These models are all largely the same, is there a way to combine them into one?

For instance, could you just have a `Post` model with a field like `parent` that relates one Post instance to another Post instance?